### PR TITLE
DRM schema versioning

### PR DESCRIPTION
**Pull request by aucuparia**

### CODE REVIEW

File: datarequest.py [hunk 1]
```diff
@@ -554,22 +556,30 @@ def datarequest_reviewers_get(ctx, request_id, pending=False):
...
+def datarequest_schema_get(ctx, schema_name, version = SCHEMA_VERSION):
     """Get schema and UI schema of a datarequest form
 
     :param ctx:         Combined type of a callback and rei struct
     :param schema_name: Name of schema
+    :param version:     Version of schema
 
     :returns: Dict with schema and UI schema
     """
+    log.write(ctx, '=======================================')
```
Comment (lwesterhof) [inline datarequest.py]: Debug statements can be removed?

File: datarequest.py [hunk 2]
```diff
@@ -66,7 +66,9 @@
 
 JSON_EXT          = ".json"
 
-SCHEMACOLLECTION  = constants.UUSYSTEMCOLLECTION + "/datarequest/schemas/youth-0"
+SCHEMACOLLECTION  = constants.UUSYSTEMCOLLECTION + "/datarequest/schemas"
+SCHEMA_URI_PREFIX = "https://yoda.uu.nl/datarequest/schemas/"
+SCHEMA_VERSION    = "youth-1"
```
Comment (lwesterhof) [inline datarequest.py:71]: In a later iteration we can make this a configuration option, so that the schema is not hard coded.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: datarequest.py (hunk 2 above)
Comment (aucuparia) [reply datarequest.py:71]: Added to backlog.

File: datarequest.py (hunk 1 above)
Comment (aucuparia) [reply datarequest.py]: Done.

**Review by aucuparia — COMMENTED** [review]
All issues solved.

**Review by lwesterhof — APPROVED** [review]
Because one of the schemas is a symlink we get an error, because the two schemas have the same name.

Don't the other schemas like assignment and datamanager review need a schema id as well?

**Comment by aucuparia** [comment]
> Because one of the schemas is a symlink we get an error, because the two schemas have the same name.
> 
> Don't the other schemas like assignment and datamanager review need a schema id as well?

Symlink is gone now, and I've added $id fields to the other schemas as well.

**Review by lwesterhof — APPROVED** [review]
LGTM

**Pull request merged**